components/StarportBanner.vue

```diff
@@ -73,9 +64,17 @@
     margin-right var(--spacing-7)
 
 .icon__left
-  color var(--white-300)
+  color var(--primary-600)
 
 @media screen and (max-width: 800px)
   .starport-banner
     grid-template-columns 1fr
+    background-size cover
+    padding-left 0
+    padding-right 0
+    opacity 0.95
```

Comment:
Added opacity here because the starport rocket logo's color is clashing with the text color. WDYT? 🤔🤔🤔 @nassdonald